.github/workflows/test-k8s-service-templates.yml

```diff
@@ -70,8 +70,13 @@ jobs:
         continue-on-error: true
         run: |
           set -Eeuo pipefail
-          # Delete empty old replicas
-          kubectl delete --namespace templates-api $(kubectl get all --namespace templates-api | grep replicaset.apps | awk '{if ($2 + $3 + $4 == 0) print $1}')
+          # Delete empty old replica sets
+          emptyReplicaSets=$(kubectl get replicasets --ignore-not-found | tail --lines=+2 | awk '{if ($2 + $3 + $4 == 0) print $1}')
+          echo -e "Found empty replica sets:"
+          echo "$emptyReplicaSets"
+          echo "-------------------------" 
+          echo "$emptyReplicaSets" | xargs --no-run-if-empty -I {} kubectl delete replicaset "{}"
+          
```

Comment:
Nepresiel mi release templates-api:
![image](https://user-images.githubusercontent.com/19371734/201327658-d0bbe22b-d246-46e0-a2c1-d4e6e40aff37.png)

Tento stript maze pred `diff` stare repliky, ktore maju vo vypise `3x 0`:
![image](https://user-images.githubusercontent.com/19371734/201327776-d61fac6e-d7c2-48b2-a18f-e2e53debf40c.png)

Chybalo tu:
- preskocenie hlavicky `tail --lines=+2`.
- ignorovanie chyby, ak sa nic nenajde `--ignore-not-found`.
- a to, ze tych replik moze byt aj viac `xargs`.
- `--namespace` nie je potrebne, vyssie vo workflow je `kubectl config set-context --current "--namespace=$KUBERNETES_NAMESPACE"`.